src/app/lmem/notice.ts

```diff
@@ -3,6 +3,16 @@ import { Ratings } from './rating';
 import { Contributor, ContributorId, NewContributor } from './contributor';
 import { captureMessage } from '../utils/sentry';
 
+export interface NoticeItem {
```

Comment:
Shouldn’t we try to merge `NoticeItem` and `BaseNotice` to one unique model ?

Reply:
Definitely! 
But can't do that until Friday though.

Reply:
Could have a second look at this, I had to update the API as well to get something neat:
> https://github.com/dis-moi/backend/pull/298